0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -236,6 +237,35 @@ p.send({
 
 For a detailed description of these properties, please see [`OutgoingTransfer`](#outgoingtransfer).
 
+#### sendMessage
+<code>ledgerPlugin.sendMessage( **message**:[OutgoingMessage](#outgoingmessage) ) ⇒ Promise.&lt;null></code>
+
+Plugin must be connected, otherwise the promise should reject. Send a ledger-local message.
+If there is a problem with the structure or validity of the message, then `sendMessage` should throw an error in the form of a rejected promise.
+
+###### Parameters
+| Name | Type | Description |
+|:--|:--|:--|
+| message | <code>[OutgoingMessage](#outgoingmessage)</code> | Properties of the message to be created |
+
+###### Returns
+**`Promise.<null>`** A promise which resolves when the message has been submitted (but not necessarily delivered.)
```

Comment:
We need to figure out at what level retries should happen. Should you assume that the message will get through if the promise resolves? Or is there a case when you'd need to retry it?